### PR TITLE
feat(kyc): Structured Address and Unstructured Address in Partner-Conducted KYC

### PR DESCRIPTION
**Pull request by magsharon**
Structured Address and Unstructured Address in Partner-Conducted KYC

Ticket Link: https://www.notion.so/immersve/Documentation-clarify-structured-Unstructured-address-13b1d446ed8a806c8a35ff29b43a23eb

### CODE REVIEW

**Comment by forktrucka** [comment]
deployed to -> https://566-merge--jovial-scone-ec740d.netlify.app/

File: imsv-docs-astro/src/content/docs/guides/kyc/partner-conducted-kyc.mdoc [hunk 1]
```diff
@@ -173,6 +173,56 @@ curl -X PUT "https://test.immersve.com/api/accounts/${cardholder_account_id}/par
   }'
 ```
 
+### **Structured Address and Unstructured Address in Partner-Conducted KYC**
```
Comment (NicBathgate) [inline imsv-docs-astro/src/content/docs/guides/kyc/partner-conducted-kyc.mdoc:176]:
```suggestion
### **Structured Address and Unstructured Address in Partner conducted KYC**
```

File: imsv-docs-astro/src/content/docs/guides/kyc/partner-conducted-kyc.mdoc [hunk 2]
```diff
@@ -173,6 +173,56 @@ curl -X PUT "https://test.immersve.com/api/accounts/${cardholder_account_id}/par
...
+
+For partner-conducted KYC, Immersve allows two formats for address input: Structured Address and Unstructured Address, used in the {% link endpoint="submit-kyc-statement" /%} endpoint. Address details are included within the claims field in the request body.
```
Comment (NicBathgate) [inline imsv-docs-astro/src/content/docs/guides/kyc/partner-conducted-kyc.mdoc:178]:
```suggestion
The {% link endpoint="submit-kyc-statement" /%} endpoint accepts two alternative address formats as part of the claims field in the request body:
- Structured Address
- Unstructured Address
```

File: imsv-docs-astro/src/content/docs/guides/kyc/partner-conducted-kyc.mdoc [hunk 3]
```diff
@@ -173,6 +173,56 @@ curl -X PUT "https://test.immersve.com/api/accounts/${cardholder_account_id}/par
...
+    "region": "Auckland",
+    "state": "AKL",
+    "postalCode": "6140",
+    "country": "NZ"
+  }
+}
+```
+
+### **Unstructured Address**
+The **Unstructured Address** is an alternative format for submitting address information. When structured addresses are not available, partners can provide an Unstructured Address. This format consolidates the address into a single text string in the fullAddress field, which Immersve later attempts to parse into structured components.
```
Comment (NicBathgate) [inline imsv-docs-astro/src/content/docs/guides/kyc/partner-conducted-kyc.mdoc:204]:
```suggestion
When it is not possible to provide a Structured Addresses, partners can provide an **Unstructured Address**. This format consolidates the address into a single text string in the `fullAddress` field, which Immersve later attempts to parse into structured address components.
```

File: imsv-docs-astro/src/content/docs/guides/kyc/partner-conducted-kyc.mdoc [hunk 4]
```diff
@@ -173,6 +173,56 @@ curl -X PUT "https://test.immersve.com/api/accounts/${cardholder_account_id}/par
...
+    "fullAddress": "2/45 Laddier Road, Centurion, The Reeds, Auckland, Auckland 0100, New Zealand",
+    "country": "NZ"
+  }
+}
+```
+
+## **Key Considerations**
+- Both Structured Address and Unstructured Address formats are supported.
+- Use Structured Address where possible: Structured addresses are preferred for accuracy and consistency.
+- Use unstructured addresses only if structured data is unavailable from the partner's IDV provider.
```
Comment (NicBathgate) [inline imsv-docs-astro/src/content/docs/guides/kyc/partner-conducted-kyc.mdoc:222]:
```suggestion
- Use Unstructured Addresses only if structured address data is not available from your IDV provider.
```

File: imsv-docs-astro/src/content/docs/guides/kyc/partner-conducted-kyc.mdoc [hunk 5]
```diff
@@ -173,6 +173,56 @@ curl -X PUT "https://test.immersve.com/api/accounts/${cardholder_account_id}/par
...
+- Immersve attempts to parse unstructured addresses into structured components after submission. Ensure the fullAddress field is as detailed as possible.
```
Comment (NicBathgate) [inline imsv-docs-astro/src/content/docs/guides/kyc/partner-conducted-kyc.mdoc:223]:
```suggestion
- Immersve attempts to parse Unstructured Addresses into structured address components after submission. Ensure the fullAddress field is as detailed as possible.
```

**Review by NicBathgate — APPROVED** [review]
Thanks Magda!

**Pull request merged**